src/components/Table/Table.scss

```diff
@@ -40,13 +40,8 @@
 
       // But show the mark.
       &::after {
-        @include ms-Icon;
-        content: '\e041';
+        content: '\e73A';
```

Comment:
The pattern we're following now is to use the Sass mixin to fill in this content. So you'd want to do `@include ms-Icon--CheckboxComposite` and that will add `content: '\E73A'.